contracts/eden/src/actions/induct.cpp

```diff
@@ -19,6 +20,8 @@ namespace eden
       {
          members.check_active_member(witness);
       }
+      if (members.is_new_member(invitee))
+         members.create(invitee);
       members.check_pending_member(invitee);
```

Comment:
this `check_pending_member` can be inside an else branch because this check after creation would be redundant

Reply:
changed